src/chanjo2/endpoints/overview.py

```diff
@@ -0,0 +1,34 @@
+from os import path
+
+from fastapi import APIRouter, Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+from chanjo2.demo import DEMO_OVERVIEW_QUERY_DATA
+from chanjo2.meta.handle_overview_content import get_overview_data
+from chanjo2.models.pydantic_models import GeneralReportQuery
+
+
+def get_templates_path() -> str:
+    """Returns the absolute path to the templates folder of this app."""
+    APP_ROOT: str = path.abspath(path.join(path.dirname(__file__), ".."))
+    return path.join(APP_ROOT, "templates")
+
+
+templates: Jinja2Templates = Jinja2Templates(directory=get_templates_path())
```

Comment:
```suggestion
templates = Jinja2Templates(directory=get_templates_path())
```
We have started not type hinting when calling a constructor as it is already clear what the type is and it improves readability.

Reply:
I agree!